spectacles/validators/content.py

```diff
@@ -154,8 +155,11 @@ def _get_errors_from_result(
                         else None
                     ),
                 )
-                if content_error not in explore.errors:
+                if explore and content_error not in explore.errors:
                     explore.errors.append(content_error)
                     content_errors.append(content_error)
+                elif model and content_error not in model.errors:
+                    model.errors.append(content_error)
+                    content_errors.append(content_error)
```

Comment:
This feels easier to read in a nested if.

```suggestion
                if content_error not in explore.errors:
                    if explore:
                        explore.errors.append(content_error)
                        content_errors.append(content_error)
                    else:
                        model.errors.append(content_error)
                        content_errors.append(content_error)
```